apps/web/src/routes/PostDetail/PostDetail.js

```diff
@@ -398,14 +398,14 @@ export function ActionCompletionsSection ({ currentUser, post }) {
   const completers = post.completionResponses.map(response => response.user)
 
   return (
-    <div className='border-border border-2 rounded-lg p-4 flex flex-row gap-2'>
+    <div className='border-border border-2 rounded-lg p-4 flex flex-row gap-2 mt-4 w-full justify-between'>
       <PeopleInfo
         people={completers}
         peopleTotal={completers.length}
         onClick={toggleCompletionResponsesDialog}
         excludePersonId={currentUser.id}
         phrases={{
-          emptyMessage: t('No one has completed this action yet'),
+          emptyMessage: t('No one has completed this - be the first!'),
```

Comment:
Changing the key for a piece of translated text just means this won't find any values in the locales and will just show the key in all instances.

Please make sure the key matches a value in both the en.json and es.json for web, and that the value represents the copy that you want